common/src/main/scala/utils/attempt/Attempt.scala

```diff
@@ -106,7 +104,7 @@ object Attempt {
     * This implementation returns the first failure in the resulting list,
     * or the successful result.
     */
-  def traverse[A, B, M[X] <: TraversableOnce[X]](as: M[A])(f: A => Attempt[B])(implicit cbf: CanBuildFrom[M[A], B, M[B]], ec: ExecutionContext): Attempt[M[B]] = {
+  def traverse[A, B, M[X] <: TraversableOnce[X]](as: M[A])(f: A => Attempt[B])(implicit cbf: BuildFrom[M[A], B, M[B]], ec: ExecutionContext): Attempt[M[B]] = {
```

Comment:
wow, special thanks due for this one as neither of these files are mentioned in the warning... think it would've taken me a while to unpick if I'd had to do it!